doc/modules/ROOT/pages/tutorials/import-sample-export-gnn.adoc

```diff
@@ -1,6 +1,6 @@
 // DO NOT EDIT - AsciiDoc file generated automatically
 
-= Sampling, Export and Integration with PyG example
+= Sampling, Export and Integration
```

Comment:
I kinda wanna have `PyG` in the title for search hits

Reply:
Putting it as the first word would make it kinda weird, no? 

```suggestion
= Pyg Sampling, Export and Integration
```

Reply:
Yes that's not right; the sampling is ours (Neo4j). It's the 'integration' that is with PyG. This could work
```suggestion
= Sampling, Export and PyG integration
```

Reply:
@Mats-SX How about "PyG integration: Sample and Export"? This would be within the 70 limit while, I think, still retaining the original meaning.

Reply:
@nvitucci OK let's do that